From bb54c3fd37ded9faa4cc88ba1a8de3e30b9b720d Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 30 Jun 2019 07:27:09 +0200
Subject: [PATCH 1/1] efi_selftest: description of CMD_BOOTEFI_SELFTEST

The current short description has a typo. Let it stand out clear that we
provide unit tests.

Improve the description of the CMD_BOOTEFI_SELFTEST configuration option.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/Kconfig | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/efi_selftest/Kconfig b/lib/efi_selftest/Kconfig
index 59f9f36801..52ad439220 100644
--- a/lib/efi_selftest/Kconfig
+++ b/lib/efi_selftest/Kconfig
@@ -1,9 +1,9 @@
 config CMD_BOOTEFI_SELFTEST
-	bool "Allow booting an EFI efi_selftest"
+	bool "UEFI unit tests"
 	depends on CMD_BOOTEFI
 	imply FAT
 	imply FAT_WRITE
 	help
-	  This adds an EFI test application to U-Boot that can be executed
-	  with the 'bootefi selftest' command. It provides extended tests of
-	  the EFI API implementation.
+	  This adds an UEFI test application to U-Boot that can be executed
+	  via the 'bootefi selftest' command. It provides extended tests of
+	  the UEFI API implementation.
-- 
2.20.1

